client/navigation/style.scss

```diff
@@ -127,4 +131,8 @@ body.is-wc-nav-folded {
 	#woocommerce-embedded-root.is-embed-loading {
 		margin-bottom: -$adminbar-height;
 	}
+
+	font > .xe-notice {
+		margin-top: $header-height;
```

Comment:
I'm afraid of going back down this path where we need to define margins on all elements, especially if they can be variable.

Maybe we should move the margin/padding I added for `#wpbody` in https://github.com/woocommerce/woocommerce-admin/pull/6247 to the `body`.  What do you think?